pulp_rpm/app/models/repository.py

```diff
@@ -427,10 +427,15 @@ def content_handler(self, path):
             repository, publication = self.get_repository_and_publication()
             if not publication:
                 return
-            base_url = f"{settings.CONTENT_ORIGIN}{settings.CONTENT_PATH_PREFIX}{self.base_path}/"
+            base_url = "{}/".format(
```

Comment:
Trailing `/` are stripped and the content app doesn't do the auto-redirect thing.

Reply:
:+1: